lib/menu.rb

```diff
@@ -0,0 +1,23 @@
+class Menu
+  
+  attr_accessor :dishes, :menu
```

Comment:
I feel like this could maybe be attr_reader? instead of accessor. 
Should :menu be :menu_items?